packages/block-library/src/avatar/index.php

```diff
@@ -0,0 +1,148 @@
+<?php
+/**
+ * Server-side rendering of the `core/avatar` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Renders the `core/avatar` block on the server.
+ *
+ * @param array    $attributes Block attributes.
+ * @param string   $content    Block default content.
+ * @param WP_Block $block      Block instance.
+ * @return string Return the avatar.
+ */
+function render_block_core_avatar( $attributes, $content, $block ) {
+	$size               = isset( $attributes['size'] ) ? $attributes['size'] : 96;
```

Comment:
Shouldn't this match [the default for the attribute specified in `block.json`](https://github.com/WordPress/gutenberg/pull/38591/files#diff-18fc06dfbdc8d315a62b79652e06072fd934c6f8bb3a95d816aba1f39926729bR15)?

Reply:
96 is the maximum provided by the WordPress `get_avatar()` function, let's update the default on the `block.json` better.